[HSEARCH] 8.0.0.Beta1 (#216)
hibernate/in.relation.to: posts/MarkoB/2025-05-11-hibernate-search-8-0-0-Beta1.adoc

Lucene (link:{hsearch-jira-url-prefix}/HSEARCH-5373[HSEARCH-5373])::
The Lucene-next backend now uses Lucene 10.2.1.
[[elasticsearch-version]]
Elasticsearch (link:{hsearch-jira-url-prefix}/HSEARCH-5361[HSEARCH-5361]/(link:{hsearch-jira-url-prefix}/HSEARCH-5365[HSEARCH-5365]::
The Elasticsearch backend works with Elasticsearch 9.0 and 8.18, as well as other already compatible versions.
[[opensearch-version]]
OpenSearch (link:{hsearch-jira-url-prefix}/HSEARCH-5363[HSEARCH-5363]::
The Elasticsearch backend works with OpenSearch 3.0, as well as other already compatible versions.
[[others-version]]
Others::
* link:{hsearch-jira-url-prefix}/HSEARCH-5374[HSEARCH-5374]: Upgrade to Elasticsearch **client** 9.0.1
* link:{hsearch-jira-url-prefix}/HSEARCH-5370[HSEARCH-5370]: Upgrade to GSON 2.13.1.Final
* link:{hsearch-jira-url-prefix}/HSEARCH-5368[HSEARCH-5368]: Upgrade to Jandex 3.3
* link:{hsearch-jira-url-prefix}/HSEARCH-5371[HSEARCH-5371]: Upgrade to Jackson 2.19.0

[[static-metamodel]]
=== Static metamodel

Hibernate Search's link:{hsearch-doc-url-prefix}#static-metamodel[static metamodel] is a set of generated classes that represents the structure of each entity's index,

> thereby allowing to reference index fields in a type-safe way to create search queries through the link:{hsearch-doc-url-prefix}#search-dsl[Search DSL].

Reviewer: Here's my regular nitpicky reminder that "allow to" is grammatically incorrect; only "allow `<somebody>` to" is correct :grin: 

```suggestion
thereby allowing type-safe references to index fields when creating search queries through the link:{hsearch-doc-url-prefix}#search-dsl[Search DSL].
```
